porch/func/go.mod

```diff
@@ -3,33 +3,42 @@ module github.com/GoogleContainerTools/kpt/porch/func
 go 1.17
 
 require (
+	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-setters v0.2.0
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/klog/v2 v2.40.1
-	sigs.k8s.io/kustomize/kyaml v0.13.2
+	sigs.k8s.io/kustomize/kyaml v0.13.3
 )
 
+replace github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-setters v0.2.0 => github.com/martinmaly/kpt-functions-catalog/functions/go/apply-setters v0.0.0-20220203180553-30cdd12b9ad5
```

Comment:
@droot - this relies on https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/757